docs/run-a-node/participate/generate_keys.md

```diff
@@ -9,25 +9,31 @@ _[Read more about how Participation Keys function in the Algorand Consensus Prot
 
 To generate a participation key, use the [`goal account addpartkey`](../../../clis/goal/account/addpartkey) command on the node where the participation key will reside. This command takes the address of the participating account, a range of rounds, and an optional key dilution parameter.  It then generates a [VRF key pair](../../../get-details/algorand_consensus#verifiable-random-function) and, using optimizations, generates a set of single-round voting keys for each round of the range specified. The VRF private key is what is passed into the VRF to determine if you are selected to propose or vote on a block in any given round. 
 
 === "goal"
     ```zsh 
-    $ goal account addpartkey -a <address-of-participating-account> --roundFirstValid=<partkey-first-round> --roundLastValid=<partkey-last-round> --keyDilution=<key-dilution-value> 
+    $ goal account addpartkey -a <address-of-participating-account> --roundFirstValid=<partkey-first-round> --roundLastValid=<partkey-last-round>
     Participation key generation successful
     ```
 
-This creates a participation key in the ledger directory of the node, which is where it should ultimately live. Use the `-o` flag to specify a different directory in the case where you will eventually transfer your key to a different node's ledger directory.
+This creates a participation key on the node. You can use the `-o` flag to specify a different location in the case where you will eventually transfer your key to a different node to construct the keyreg transaction.
 
 !!! tip
-	To optimize storage, the Key Dilution parameter defaults to the square root of the participation period length. It determines how many ephemeral keys will be stored on an algorand node, as they are generated in batches. For example, if your participation period is set to 3,000,000 rounds, a batch of 1,732 ephemeral keys will be generated upfront, with additional batches getting generated after each set is used.
+	To optimize storage, the Key Dilution parameter defaults to the square root of the participation period length but this can be overridden with the flag `--keyDilution`. The Key Dilution determines how many ephemeral keys will be stored on an algorand node, as they are generated in batches. For example, if your participation period is set to 3,000,000 rounds, a batch of 1,732 ephemeral keys will be generated upfront, with additional batches getting generated after each set is used.
+
+# Add participation key via API
 
-# Check that the key exists in the node's ledger directory
+If you chose to save the participation key and now want to add it to the server, you can use the following API call to add the partkey file to the node.
+
+```bash
+$ curl -X POST $URL/v2/participation -H "X-Algo-API-Token: $token" --data-binary @ALICE...VWXYZ.0.30000.partkey
+```
+Upon successfully calling the endpoint you will be returned the Participation ID (`partId`), otherwise an error will be returned.
```

Comment:
Do this with goal, not curl:
`goal account installpartkey --partkey ALICE...VWXYZ.0.30000.partkey --delete-input -d your-data-directory`